debezium-testing/debezium-testing-system/src/main/java/io/debezium/testing/system/tools/kafka/DockerKafkaController.java

```diff
@@ -51,7 +57,8 @@ public String getTlsBootstrapAddress() {
     @Override
     public boolean undeploy() {
         container.stop();
-        return container.isRunning();
+        zookeeperContainer.stop();
+        return zookeeperContainer.isRunning();
```

Comment:
```suggestion
        return container.isRunning() && zookeeperContainer.isRunning();
```
but I am a bit confused on te return value, it seems to me accroding to JavaDoc of the method on interface it should return negated value of `isRunning()` as I suppos that successful `undeploy` means that container is NOT running. If it is the case it should be fixed at other controllers too.

Reply:
Great catch with the JavaDoc discrepancy! I will fix all the controllers accordingly.